added asssets and forcefield

belletinsley opened the pull request:
it only works in level 5

all the assets are in tinsley pieces

belletinsley committed:
i think this works

belletinsley committed:
Merge branch 'main' of https://github.com/Dominic-Roser/Row-mageddon into tinzbranch

belletinsley committed:
more of the merge

belletinsley committed:
updated forcefield

belletinsley committed:
added assets and made forcefield

belletinsley committed:
Merge branch 'main' of https://github.com/Dominic-Roser/Row-mageddon into tinzbranch

belletinsley committed:
something about player data: plz dont break anything this isn't loading

Commit:
Revert "added assets and made forcefield"

This reverts commit 63f9db2c96f2ad6de1c835150e74e1b0e9546a16.

belletinsley committed:
Merge branch 'main' of https://github.com/Dominic-Roser/Row-mageddon into tinzBranch

belletinsley committed:
added assets

belletinsley committed:
added forcefield

Sean0z reviewed (changes requested):
- the power up itself works great
-  On level 5 when the enemy crosses over the checkpoint, there is a null reference exception at checkpoint.cs 
- reset player data so that you're level 0 and dont own any power ups
- There is a white box in my owned boats
- reduce the time that the force field is active

belletinsley committed:
- reset player data so that you're level 0 and dont own any power ups
- there is a white box in my owned boats
reduce the time that the force field is active
